applications/DEMApplication/python_scripts/sphere_strategy.py

```diff
@@ -495,14 +495,6 @@ def RotationalIntegrationSchemeTranslator(self, name_translational, name_rotatio
 
         if name_rotational == 'Direct_Integration' or name_rotational == 'same_as_translational':
             class_name = self.TranslationalIntegrationSchemeTranslator(name_translational)
-        elif name == 'Forward_Euler':
```

Comment:
My mistake, @farrufat-cimne . Thanks for fixing. However, the error was to put 'name'' instead of ''name_rotational'.
We should give the option of choosing "Velocity_Verlet" for translational and "Forward_Euler" for rotational. So we cannot delete all those options, they must be fixed by switching 'name' to 'name_rotational'.